doc/cascadia/profiles.schema.json

```diff
@@ -873,11 +873,29 @@
             "string",
             "null"
```

Comment:
As mentioned [here](https://github.com/microsoft/terminal/issues/7443#issuecomment-682529863), `name` can also be an object in `{ "key": string }` format.

This one is just bonus points though. We use this in defaults.json to get a localized version of an action's name. I don't really expect people to use this in their settings.json (but they can if they want to).